doc/on_chain_spec.md

```diff
@@ -526,6 +526,11 @@ inform prospective bidders in the auction announcement
 how much ADA they should deposit to qualify for the auction.
 However, none of this is enforced by any of the auction’s on-chain validators.
 
+Losing bidders can collect their deposit at any time after the bidding end time.
```

Comment:
But Standing Bid can be cleaned after cleanup time and that means that `BidderDeposit` tx would not work.